src/pages/_app.js

```diff
@@ -1,9 +1,16 @@
 import App from 'next/app';
+import { Provider } from 'react-redux';
+
+import store from '@redux/store';
 
 class MyApp extends App {
```

Comment:
I'm curious about how we have an App.js, an _app.js, and an `import App from 'next/app`. Perhaps out of scope for this PR, but I wonder if you have any ideas for deconflicting these.

Reply:
`_app.js` is a special file in Next that we unfortunately can't rename or change too much other than the class. `components/App.js` is moot now that I moved the code into the `pages/index.js`